omnibus/files/private-chef-ctl-commands/cleanup.rb

```diff
@@ -4,13 +4,10 @@
 add_command_under_category "cleanup", "general", "Perform post-upgrade removal of now-obsolete data, configuration files, logs, etc.  Add the '--no-op' flag to see what *would* be removed.", 2 do
   use_why_run_mode = ARGV.include?("--no-op")
 
-  # Our cleanup process is really just a special chef run
-  command = ["chef-client -z",
-             "--config #{base_path}/embedded/cookbooks/solo.rb",
-             "--json-attributes #{base_path}/embedded/cookbooks/post_upgrade_cleanup.json",
-             "--log_level fatal"] # yes, that's an underscore in log_level
-  command << "--why-run" if use_why_run_mode
+  chef_args = ""
+  chef_args << "--why-run" if use_why_run_mode
```

Comment:
We seem to have lost setting the log_level to fatal. Not sure if that was intentional or not. Looking at the run_chef code in omnibus ctl I'm not seeing the log level set anywhere, so I'm assuming it's just using the default in this case. I'm not sure what the default is however.